src/cart/components/CartItem.jsx

```diff
@@ -37,17 +27,17 @@ export const CartItem = ({
   setQuantity
 }) => {
   return (
-    <div className='row cart-item'>
+    <div className='row cart-item text-md-left text-center'>
       <div className='col-lg-3 col-12'>
         <img className='img-fluid' src={image} alt='' />
       </div>
       <div className='col-lg-9 col-12 pt-3'>
         <h4 className='product-title'>{title}</h4>
         <span className='product-price'>{numberWithCommas(price)}</span>
-        <CartItemCount id={id} setQuantity={setQuantity} value={quantity} />
+        <CartItemCount id={id} setQuantity={setQuantity} quantity={quantity} />
         <p className='product-des'>{description}</p>
         <button
-          onClick={() => removeItem(id)}
+          onClick={() => removeItem(id, false)}
```

Comment:
the `removeItem` function only take 1 argument. I'm not sure why it needs two here ?